configargparse.py

```diff
@@ -893,7 +894,7 @@ def print_values(self, file = sys.stdout):
         """Prints the format_values() string (to sys.stdout or another file)."""
         file.write(self.format_values())
 
-    def format_help(self):
+    def _config_file_help(self):
```

Comment:
Would rather not split or rename this method, especially since it adds help text for env. vars and not just config files.

Reply:
Reverted with commit 2205855.